python/lsst/ts/standardscripts/auxtel/enable_atcs.py

```diff
@@ -0,0 +1,79 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["EnableATTCS"]
+
+import yaml
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.auxtel.atcs import ATCS, ATCSUsages
+
+
+class EnableATTCS(salobj.BaseScript):
+    """ Enable all ATCS components.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    **Details**
+
+    """
+
+    __test__ = False  # stop pytest from warning that this is not a test
+
+    def __init__(self, index):
+        super().__init__(index=index, descr="Enable ATCS.")
+
+        self.attcs = ATCS(self.domain, intended_usage=ATCSUsages.StateTransition)
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/auxtel/EnableATTCS.yaml
+            title: EnableLATISS v1
+            description: Enable ATCS. Does not require any configuration.
+            type: object
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
```

Comment:
This may be a first, but I'm wondering if you really need this doc string, especially since this script has no configuration? In theory sphinx can link to the parent class in salobj, but perhaps we don't have that linking working yet. (if you remove it here you might look at your other new scripts for the same).